omdb-client/impl/src/main/java/com/garret/movies/omdb/impl/config/OmdbConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.garret.movies.omdb.impl.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Configuration
+@PropertySource("classpath:/omdb.properties")
+public class OmdbConfig {
+
+    @Bean
+    public RestTemplate restTemplate() {
```

Comment:
This is regular RestTemplate and does not contains any special for OMDB and in addition you already have bean for rest template in context provided by Spring Boot AutoConfiguration

Reply:
@mmarusich I need http converter which is not in default rest template, that's why i configure it